elfpy/markets/hyperdrive/hyperdrive_actions.py

```diff
@@ -2197,3 +2153,19 @@ def calc_free_margin_fp(
         withdraw_pool_deltas.withdraw_interest = interest
     # Finally return the amount used by this action and the caller can update reserves.
     return withdraw_pool_deltas
+
+
+def check_output_assertions(trade_result: trades.TradeResultFP):
```

Comment:
this is basically `check_output_assertions_FP` right? why not put it in base pricing model next to the existing `check_output_assertions`? once we switch over, you'll have to track it down anyway.

Reply:
We are no longer going to call it from the pricing model. Instead it'll be a local function available from hyperdrive_actions (i.e. doesn't belong to any class). So the version in the pricing model will be deleted once I purge float from the repo.